src/parser.lua

```diff
@@ -0,0 +1,37 @@
+local parser = {}
+
+function parser.packLocation(input)
```

Comment:
I put functions that {Scene,Structure}Parser use in parser.lua. If you can think of a better name, let me know. Pack? `pack.location()`, `pack.data()`? I don't know.

Reply:
I think the function names are fine. I don't like calling it "parser". It seems too generic. I think it should age before choosing a different name though.

Reply:
It is too generic, and it's not quite accurate. It isn't a parser, it's functions used by the parser.